Makefile

```diff
@@ -55,8 +55,7 @@ clean:
 run-python-test: $(COMPILED_TESTS)
 	PYENV_VERSION=pypy3.7-7.3.9 . cairo-rs-py-env/bin/activate && \
 	maturin develop && \
-	python3 hints_tests.py && \
-	python3 get_builtins_initial_stack.py && \
+	./run_python_tests.sh && \
```

Comment:
Is the `PYENV_VERSION` still the same here? I'm not entirely sure if the `venv` takes care of that.

Reply:
As far as i know yes, venv takes care about the python version with which the tests are ran.